### PR TITLE
FUSETOOLS-3512: Fix UI Test 'CamelEditorDataFormatTest' and convert i…

### PR DESCRIPTION
**Pull request by pospisilf**
…nto master branch

# Pull Request Checklist

After this checklist is all checked or PR provides explanations for possible pass-through, please put the label "Ready for review" on the PR.

## General

- [ ] Did you use the Jira Issue number in the commit comments?
- [ ] Did you set meaningful commit comments on each commit?
- [ ] Did you sign off all commits for this PR? (git commit -s -m "jira issue number - your commit comment")

## Functional

- [ ] Did the CI job report a successful build?
- [ ] Is the non-happy path working, too?

## Maintainability

- [ ] Are all Sonar reported issues fixed or can they be ignored?
- [ ] Is there no duplicated code?
- [ ] Are method-/class-/variable-names meaningful?

## Tests

- [ ] Are there unit-tests?
- [ ] Are there integration tests (or at least a jira to tackle these)?
- [ ] Do we need a new UI test?

## Legal

- [ ] Have you used the correct file header copyright comment?
- [ ] Have you used the correct year in the headers of new files?

### CODE REVIEW

**Comment by pospisilf** [comment]
testPR

**Comment by pospisilf** [comment]
testPR

File: uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/CamelEditorDataFormatTest.java [hunk 1]
```diff
@@ -96,6 +98,29 @@ public CamelEditorDataFormatTest(String dataFormat) {
...
+	/**
+	 * Checks if used Camel Version is newer than input.
+	 *
+	 * @param version
+	 *                    Integer representing required version for true return. f.e. 7 -> 7.0.0, 73 -> 7.3.0, 731 ->
+	 *                    7.3.1
+	 * @return true if actual version is equal to required or newer
+	 * @return false if actual version is older then required
+	 */
+	private static boolean IsCamelVersionNewerThan(Integer version) {
```
Comment (apupier) [inline uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/CamelEditorDataFormatTest.java:110]: I'm wondering if org.osgi.framework.Version is not available and in this case, there might be a shorter solution. I think it worth checking

Comment (pospisilf) [reply uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/CamelEditorDataFormatTest.java:110]: @apupier Thanks for tip, I will look at this.

**Comment by pospisilf** [comment]
testPR

**Review by apupier — APPROVED** [review]

**Review by djelinek — APPROVED** [review]

**Pull request merged**